src/block/block_open.c

```diff
@@ -369,7 +369,7 @@ __desc_read(WT_SESSION_IMPL *session, WT_BLOCK *block)
 		WT_ERR_MSG(session, WT_ERROR,
 		    "unsupported WiredTiger file version: this build only "
 		    "supports major/minor versions up to %d/%d, and the file "
```

Comment:
I think this is correct as written, those are printing defined constants, WT_BLOCK_MAJOR_VERSION and WT_BLOCK_MINOR_VERSION.